main.tex

```diff
@@ -295,11 +296,11 @@ \section*{Abstract} %(301 out of 300 words)
 \section*{Author summary}
 
 We revise a provenance ontology for simulation studies of cellular biochemical models.
-Provenance information is useful for understanding the creation of a simulation model, because it does not only contain information about the entities and activities that have led to a simulation model but also the relations of these, which can be visualized.
-It provides additional structure as research questions, assumptions, and requirements are singled out and explicitly related along with data, qualitative models, simulation models, and simulation experiments through a small set of predefined but extensible activities.
+Provenance information is useful for understanding the creation of a simulation model because it not only contains information about the entities and activities that have led to a simulation model, but also the relations, all of which can be visualized.
+It provides additional structure because research questions, assumptions, and requirements are singled out and explicitly related along with data, qualitative models, simulation models, and simulation experiments through a small set of predefined but extensible activities.
```

Comment:
"related along with data, qualitative models, simulation models, and simulation experiments through a small set of predefined but extensible activities" could be reworded maybe? I'm not 100% sure what this sentence is trying to say

Reply:
What about?
It provides additional structure by explicitly recording research questions, assumptions, and requirements relating these along with data, qualitative models, simulation models, and simulation experiments by a small set of predefined but extensible activities.
What I wish to say: we need to explicitly collect information about the research questions, assumptions, and requirements that lead to a simulation model. We can then relate these entities together with the other entities (data, qualitative model, simulation model, sim. experiment) in conjecture with pre-defined (but extensible) activities.

Reply:
Yes that sounds good! Now that I'm re-reading it, it makes more sense!